uberfire-extensions/uberfire-layout-editor/uberfire-layout-editor-api/src/main/java/org/uberfire/ext/layout/editor/api/editor/LayoutTemplate.java

```diff
@@ -26,7 +26,7 @@
 @Portable
 public final class LayoutTemplate {
 
-    private int version = 2;
+    private int version = 3;
```

Comment:
We need that on the form definition aswell!

Reply:
Hello @pefernan  - you mean we should add this field to FormDefinition and start counting from 1?

Reply:
@jesuino sorry, this comment isn't a change suggestion just a comment :D